examples/dynamo_integration/scripts/plot_throughput_histograms_per_request.py

```
@@ -0,0 +1,858 @@
+#!/usr/bin/env python3
+"""
+Histogram plotting script for throughput metrics distribution analysis.
+
+This script creates histograms showing the distribution of throughput metrics,
+similar to plot_throughput_vs_tsq_per_request.py but with count on Y-axis instead of TSQ.
+
+Features:
+- Per-LLM-call histograms (TTFT, ITL, Throughput) showing distribution of every individual LLM call
+- Per-request aggregate histograms (Total Tokens, LLM Calls, Duration)
+- Statistical annotations: median lines, ±2σ bounds, mean, std, percentiles
```

Ruff (0.14.8)

Docstring contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)